docs/UserGuide.md

```diff
@@ -236,3 +248,17 @@ _Details coming soon ..._
 | **MarkRoomsUnclean** | `markRoomsUnclean`                                                                                                                                                                             |
```

Comment:
Should we add spacing for this MarkRoomsUnclean as it is under "Action"?

Reply:
Fixed!